db/migrate/20221018225259_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :user, null: false, foreign_key: true
+    end
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.